sample_code/ep055/rev07/t.py

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
```

Comment:
Added additional examples about other things that can go before the `__future__` import like the shebang and the magic coding comment.

Reply:
idk if the coding comment is worth mentioning, since it's basically obsolete

Reply:
Yeah, what I found in the docs is that it's still supported, so you can deviate from the default utf-8 encoding if needed. Maybe I can change the `utf-8` value to something else, or do you prefer to remove that line?

Reply:
I'd prefer to remove it, you an use any old comment though -- that's also allowed before